packages/babel-helpers/src/helpers/regenerator.ts

```diff
@@ -166,389 +171,223 @@ export default function /* @no-mangle */ _regenerator() {
   function makeInvokeMethod(
     innerFn: Function,
     self: unknown,
-    context: Context,
+    tryLocsList: TryLocs[],
   ) {
     var state = GenState.SuspendedStart;
 
-    return function invoke(method: "next" | "throw" | "return", arg: any) {
+    function invoke(
+      _methodName: "next" | "throw" | "return",
```

Comment:
Do we need to keep the `_methodName` parameter here?

Reply:
It's not used, but we still keep the first argument as a string name for the sake of the shared defineIteratorMethods .